Lib/_pyrepl/windows_eventqueue.py

```diff
@@ -0,0 +1,119 @@
+"""
+Windows event and VT sequence scanner, similar to `unix_eventqueue.py`
+"""
+
+from collections import deque
+
+from . import keymap
+from .console import Event
+from .trace import trace
+import os
+
+# Reference: https://learn.microsoft.com/en-us/windows/console/console-virtual-terminal-sequences#input-sequences
+VT_MAP: dict[bytes, str] = {
+    b'\x1b[A': 'up',
+    b'\x1b[B': 'down',
+    b'\x1b[C': 'right',
+    b'\x1b[D': 'left',
+    b'\x1b[1;5D': 'ctrl left',
+    b'\x1b[1;5C': 'ctrl right',
+
+    b'\x1b[H': 'home',
+    b'\x1b[F': 'end',
+
+    b'\x7f': 'backspace',
+    b'\x1b[2~': 'insert',
+    b'\x1b[3~': 'delete',
+    b'\x1b[5~': 'page up',
+    b'\x1b[6~': 'page down',
+
+    b'\x1bOP':    'f1',
+    b'\x1bOQ':    'f2',
+    b'\x1bOR':    'f3',
+    b'\x1bOS':    'f4',
+    b'\x1b[15~':  'f5',
+    b'\x1b[17~]': 'f6',
+    b'\x1b[18~]': 'f7',
+    b'\x1b[19~]': 'f8',
+    b'\x1b[20~]': 'f9',
+    b'\x1b[21~]': 'f10',
+    b'\x1b[23~]': 'f11',
+    b'\x1b[24~]': 'f12',
```

Comment:
I may not fully understand what's going on here, but I believe these encodings are incorrect due to the stray `]`.

```suggestion
    b'\x1b[17~': 'f6',
    b'\x1b[18~': 'f7',
    b'\x1b[19~': 'f8',
    b'\x1b[20~': 'f9',
    b'\x1b[21~': 'f10',
    b'\x1b[23~': 'f11',
    b'\x1b[24~': 'f12',
```

Verified empirically with `showkey -a`; <kbd>F6</kbd> and <kbd>F12</kbd> pictured:

![image](https://github.com/user-attachments/assets/b52d27df-8844-4063-8904-f84b99b0c181)

Reply:
oh good eye, he's absolutely right about this